### ISSUE TITLE
Allow us to choose download folder location at the start

### ISSUE THREAD

**Issue by MaKTaiL** [davidmhewitt/torrential#64]
Instead of going to one specific folder everytime.

**Comment by davidmhewitt** [comment]
I think I would prefer not to implement this for the sake of keeping the user experience and interface simpler.

But, can you provide more details about why you think this is required so I can have a think of ways to improve the specific issue you're having with all torrents going into the same folder?

**Comment by MaKTaiL** [comment]
I download a lot of tv show episodes with torrent (that's basically 90% of my torrent usage). I like to organize each tv show in different folders so I must move the downloaded files everytime. Having a default download folder is fine, but we should have an option to change it for every single download if one so desires. 

Also, I see the following has been posted before but allowing us to select which files we want to download would be awesome too and it would be nicely integrated with the ability to choose the download folder imo.

**Comment by valkirilov** [comment]
+1

I would like to use Torrential when this feature is available because I really like to organize my files and the lack of this options is the only reason why I still use Transmission instead of your great work.

**Comment by selwynorren** [comment]
Yup I agree with this feature. I love the look and feel of Torrential just such a pity I cannot choose the download location. This means extra work for me to keep files and folders organised.

**Comment by OneTruthPrevails** [comment]
Interface is quite simple and intuitive, but this is one feature I believe is extremely important while managing downloads

**Comment by o-alquimista** [comment]
> But, can you provide more details about why you think this is required so I can have a think of ways to improve the specific issue you're having with all torrents going into the same folder?

I download music and movies to `~/Music` and `~/Videos`, respectively. And music albums are further organized by artist and album name (e.g. `~/Music/Chelsea Wolfe/Abyss`).

I keep seeding them forever.

**Comment by MaKTaiL** [comment]
> > But, can you provide more details about why you think this is required so I can have a think of ways to improve the specific issue you're having with all torrents going into the same folder?
> 
> I download music and movies to `~/Music` and `~/Videos`, respectively. And music albums are further organized by artist and album name (e.g. `~/Music/Chelsea Wolfe/Abyss`).
> 
> I keep seeding them forever.

Damn, this is still an issue after all these years? I didn't even remember opening this ticket back in 2017 😅.
